guides/README.md

```diff
@@ -4,36 +4,32 @@ description: Angle Protocol Guides
 
 # 🎓 Guides
 
-This section contains guides to interact with the Angle Protocol.
+This section contains guides to interact with the Angle App.
```

Comment:
Here it's the Angle Protocol since we have guides which are not directly related to the app